src/v2/api/services/DisciplineTeacherService.spec.ts

```diff
@@ -0,0 +1,207 @@
+import { Test } from '@nestjs/testing';
+import { DisciplineTeacherService } from './DisciplineTeacherService';
+import { DisciplineRepository } from '../../database/repositories/DisciplineRepository';
+import { TeacherRole } from '@prisma/client';
+import { PollService } from './PollService';
+import { DateModule } from '../../utils/date/DateModule';
+import { PrismaModule } from '../../modules/PrismaModule';
+import { TelegramAPI } from '../../telegram/TelegramAPI';
+import { MapperModule } from '../../modules/MapperModule';
+import { PollModule } from '../../modules/PollModule';
+import { ConfigurationModule } from '../../modules/ConfigModule';
+import Configuration from '../../config/Configuration';
+import { TelegramConfigService } from '../../config/TelegramConfigService';
+import { NotEnoughAnswersException } from '../../utils/exceptions/NotEnoughAnswersException';
+import { ExcessiveAnswerException } from '../../utils/exceptions/ExcessiveAnswerException';
+import { WrongTimeException } from '../../utils/exceptions/WrongTimeException';
+
+describe('DisciplineTeacherService', () => {
+  let disciplineTeacherService: DisciplineTeacherService;
+  let disciplineRepository: DisciplineRepository;
+  let pollService: PollService;
+  beforeEach(async () => {
+    const testingModule = await Test.createTestingModule({
```

Comment:
There are so many modules, which are not used during the tests. You should use .useMocker() method to avoid so many dependencies in module.